app-modules/student-data-model/src/Filament/Resources/StudentResource.php

```diff
@@ -96,6 +109,7 @@ public static function getGloballySearchableAttributes(): array
     public static function getGlobalSearchResultDetails(Model $record): array
     {
         return array_filter([
+            'Score' => $record->score,
```

Comment:
Why is Score added here but not in the one for Prospect?

Reply:
This is my bad, it was remaining from testing